NGLess/Interpretation/Count/RefSeqInfoVector.h

```diff
@@ -33,7 +33,7 @@ struct PoolC {
     private:
 
     std::vector<void*> data_;
-    int free_;
+    unsigned int free_;
```

Comment:
Actually, I think `unsigned int` is probably not the optimal option. `size_t` or something is better.

Reply:
Using `size_t` now.